src/core_atmosphere/physics/physics_wrf/module_cu_gf_deep.F

```diff
@@ -38,7 +38,9 @@ MODULE module_cu_gf_deep
 !> still 16 ensembles for clousres
      integer, parameter:: maxens3=16
 !>  which coldpool version to use
-     integer, parameter:: icoldpool=1
+     integer, parameter:: icoldpool=1  ! 1 = momentum impact; 2 = sf flx impact
+                                       ! 3 = both, (1) and (2)
```

Comment:
clearer to put 2 on its own line

Reply:
Updated following your suggestions. Thanks.